packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -744,6 +749,15 @@ class RenderParagraph extends RenderBox
     return _textPainter.getOffsetForCaret(position, caretPrototype);
   }
 
+  /// Returns the tight bounded height of the glyph at the given [position].
+  ///
+  /// Valid only after [layout].
```

Comment:
Any of these docs comments that were copied from text_painter.dart should use a macro instead.

Here's an example of how to do that:

https://github.com/flutter/flutter/blob/0bb1e573fce0a32c28d1bd3caf26e53405604f48/packages/flutter/lib/src/painting/text_painter.dart#L335-L337

https://github.com/flutter/flutter/blob/0bb1e573fce0a32c28d1bd3caf26e53405604f48/packages/flutter/lib/src/rendering/paragraph.dart#L269

Reply:
Thanks for the examples!